src/runOnPullRequest.js

```diff
@@ -23,6 +23,11 @@ import {
 } from './constants';
 import type {NameToLabelToFiles} from './utils';
 
+// NOTE: We don't want to run Gerald when we're merging a PR into
+// master or main, as this is likely a deploy branch that only contains
+// landed PRs that have already been reviewed.
+const IGNORED_PR_REFS = ['master', 'main'];
```

Comment:
I don't think we want to do this here, since many repos do land straight to `main` / `master`. Instead, we can control this repo-by-repo with their own yaml files that call Gerald.